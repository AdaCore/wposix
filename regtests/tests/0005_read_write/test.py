from test_support import *

gnatmake('rw_test');
run('rw_test')
