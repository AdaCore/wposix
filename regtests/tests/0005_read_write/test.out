Ok, file present.
Ok, proper content found.
Ok append, proper content found.
